.github/workflows/rust.yml

```diff
@@ -218,3 +218,20 @@ jobs:
       - name: Stop docker-compose
         working-directory: ./docker
         run: docker-compose down
+
+  docs:
+    name: cargo-doc
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
```

Comment:
@janpetschexain But then we should probably also adjust here, to install the nightly toolchain, shouldn't we? Or we can still install the stable toolchain and just call `cargo doc` with the +nightly flat?

Reply:
we have to install nightly, similar to the `cargo-fmt` job.